modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
+separate upgrade strategies and are also used to control the cadence of updates.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
 4.5 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
+the next minor version of {product-title}. Upgrade Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-{product-title} {product-version}, which includes the upgrade from the previous 4.3
-release, has three upgrade channels to choose from:
+The updates within an Upgrade Channel are designated as one of two types. These
+Update Types are defined as:
 
-* candidate-{product-version}
-* fast-{product-version}
-* stable-{product-version}
+.  General Availability (GA) Software - {product-title} GA releases are fully
+supported and production quality.
+
+.  Release Candidate (RC) Software - {product-title} RC releases are not supported
+and should only be used to test feature acceptance and assist in qualifying the next
+version of {product-title} in your infrastructure. RC releases contain all the
+features of the product and are representative of an eventual GA release.
 
-The upgrade channels contain two types of updates:
-
-.  General Availability Software (or GA) - These versions of {product-title} are
-fully supported and are considered production quality. You may upgrade to the general
-availability release from either of the fast and stable channels.
-
-.  Release Candidate Software (or RC) - These versions of {product-title} are
-representative of the eventual general availability release and are available only
-in the candidate-{product-version} channel. The release candidate will contain all
-the features of the product. You are allowed to upgrade from a release candidate to
-another release candidate and to upgrade from a previous minor version of
-{product-title} to the current release candidate. Release candidate builds are not
-supported by Red Hat and you will not be able to upgrade from a release candidate to
-the general availability release of {product-title}. Candidates should be used to
-test feature acceptance and assist in qualifying the next version of
-{product-title} in your infrastructure.
 +                                                                                    
 [NOTE]                                                                               
 ==== 
-Release candidates differ from the nightly builds found on try.openshift.com. You
+RC releases differ from the nightly builds found on try.openshift.com. You
 cannot upgrade nightly builds to nightly builds. Nightly builds are available for
```

Comment:
You might be able to update between nightlies, or you may not.  How about:

> Updating to or from nightly builds is neither recommended nor supported.

Reply:
done